docs/v1/P4Runtime-Spec.mdk

```diff
@@ -483,7 +491,7 @@ number of reasons, the most likely of which are:
    protect intellectual property or maintain security.
 
 2. The target was not implemented using P4 code to begin with, although it still
-   obeys the "contract" specified in the P4Info.
+   obeys the control plane API specified in the P4Info.
```

Comment:
Do you want to emphasize here that in the absence of a P4 program, the datapath knowledge required to write the controller has to come from somewhere else: documentation, RFCs, ...

Reply:
Sure, I  will add a sentence on that with another commit, and see about changing the text wrap to 80 characters like the rest of the document.  72 is the default I have set in my Emacs config (or perhaps that comes with it by default).

Reply:
Done.